easybuild/easyconfigs/b/Boost/Boost-1.74.0-gompi-2020b.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.74.0'
+
+homepage = 'https://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://dl.bintray.com/boostorg/release/%(version)s/source/']
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['afff36d392885120bcac079148c177d1f6f7730ec3d47233aa51b0afa4db94a5']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.5'),
+]
+
+configopts = '--without-libraries=python'
```

Comment:
```suggestion
configopts = '--without-libraries=python,mpi'
```

Reply:
or rather, we should probably do `boost_mpi = False` ?

Reply:
`boost_mpi = False` should indeed be preferred and my plan with https://github.com/easybuilders/easybuild-easyblocks/pull/2167 is that `--without-libraries=python` should not be needed if Python is already not present as a dependency.